autosklearn/automl.py

```diff
@@ -235,7 +237,8 @@ def fit_on_datamanager(self, datamanager, metric):
 
     def _get_logger(self, name):
         logger_name = 'AutoML(%d):%s' % (self._seed, name)
-        setup_logger(os.path.join(self._backend.temporary_directory, '%s.log' % str(logger_name)))
+        setup_logger(os.path.join(self._backend.temporary_directory, '%s.log'
+                                  % str(logger_name)), self.log_config)
```

Comment:
Please move the 2nd argument to a new line and add a trailing comma and put the closing bracket to a newline, too.